crates/bevy_render/src/view/window/mod.rs

```diff
@@ -193,7 +195,8 @@ fn extract_windows(
 }
 
 struct SurfaceData {
-    surface: wgpu::Surface,
+    // TODO: what lifetime should this be?
```

Comment:
Leaving a note that we have to figure this out.

Reply:
I think `'static` is what we want for now until we rewrite `bevy_winit` to use safe winit APIs and surface creation.